{
    "id": "CVE-2019-5010",
    "product": "Python",
    "version": "Python.org CPython 2.7.11 Python.org CPython 3.6.6 Python.org CPython 3.5.2 Python.org CPython 3 master at 480833808e918a1dcebbbcfd07d5a8de3c5c2a66 ",
    "vulnerability": [
        "CWE-476: NULL Pointer Dereference"
    ],
    "description": "An exploitable denial-of-service vulnerability exists in the X509 certificate parser of Python.org Python 2.7.11 / 3.6.6. A specially crafted X509 certificate can cause a NULL pointer dereference, resulting in a denial of service. An attacker can initiate or accept TLS connections using crafted certificates to trigger this vulnerability.",
    "poc": {
        "reference": [
            "https://talosintelligence.com/vulnerability_reports/TALOS-2019-0758"
        ],
        "github": [
            "https://github.com/0xT11/CVE-POC",
            "https://github.com/BSolarV/cvedetails-summary",
            "https://github.com/JonathanWilbur/CVE-2019-5010",
            "https://github.com/developer3000S/PoC-in-GitHub",
            "https://github.com/hectorgie/PoC-in-GitHub",
            "https://github.com/nomi-sec/PoC-in-GitHub"
        ]
    }
}